java/code/src/com/suse/manager/utils/SaltUtils.java

```diff
@@ -1953,6 +1955,54 @@ public void updateSystemInfo(JsonElement jsonResult, String minionId) {
         });
     }
 
+    public boolean updateMaster(MinionServer minion, String master) {
```

Comment:
I would choose different name. When later UI part is implemented, we will want to trigger to actually change master on the salt client and this naming will get confusing. What is wrong with `updateProxyPath`?

Reply:
Or `updateMinionConnecitonPath`, since we are not updating the proxy but using proxy information to update the minion.